src/Shared/UnitTests/Vacancies.Client/Application/CommandHandlers/RejectVacancyCommandHandlerTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using Esfa.Recruit.Vacancies.Client.Application.CommandHandlers;
+using Esfa.Recruit.Vacancies.Client.Application.Commands;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Domain.Events;
+using Esfa.Recruit.Vacancies.Client.Domain.Messaging;
+using Esfa.Recruit.Vacancies.Client.Domain.Repositories;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Esfa.Recruit.Vacancies.Client.UnitTests.Vacancies.Client.Application.CommandHandlers
+{
+    public class RejectVacancyCommandHandlerTests
+    {
+        private readonly Mock<ILogger<RejectVacancyCommandHandler>> _mockLogger = new Mock<ILogger<RejectVacancyCommandHandler>>();
+        private readonly Mock<IVacancyRepository> _mockVacancyRepository = new Mock<IVacancyRepository>();
+        private readonly Mock<IMessaging> _mockMessaging = new Mock<IMessaging>();
+
+        [Theory]
+        [InlineData(VacancyStatus.Review)]        
+        public async Task WhenVacancyIsNotInValidState_ShouldSetVacancyReferred(VacancyStatus status)
```

Comment:
should this be WhenVacancy**IS**inValidState

Reply:
changed the text.